src/inmanta/moduletool.py

```diff
@@ -832,10 +835,10 @@ def v1tov2(self, module: str) -> None:
         """
         Convert a V1 module to a V2 module in place
         """
-        module = self.get_module(module)
```

Comment:
Just curious, why do you prefer `mod` over `module`? I tend to write it full out.

Reply:
because I needed something else than module which is a method argument of type string. I can use module_object if you prefer?

The parameter should have been module_name but that is a potential breaking change

Reply:
![image](https://github.com/inmanta/inmanta-core/assets/10819/bbf55b4e-ce47-4745-9dac-ee955728b113)